build.sbt

```diff
@@ -139,11 +139,14 @@ lazy val frc = crossProject(JVMPlatform, NativePlatform).crossType(CrossType.Ful
 lazy val frcJVM = frc.jvm
 lazy val frcNative = frc.native
 
-lazy val config = project.dependsOn(coreJVM).settings(
+lazy val config = crossProject(JSPlatform, JVMPlatform, NativePlatform).crossType(CrossType.Pure).dependsOn(core).settings(
```

Comment:
You need to copy the nativeSettings call as in other cross projects here. That should fix the failing build.

Reply:
Hmm... That didn't seem to work.

Reply:
Oh! You've added jvmDependencies in your settings, that will need to go.